src/migrate/azext_migrate/custom.py

```diff
@@ -0,0 +1,184 @@
+                          resource_group_name,
+                          project_name,
+                          group_name,
+                          machines,
+                          e_tag=None):
+    return client.create(resource_group_name=resource_group_name, project_name=project_name, group_name=group_name, e_tag=e_tag, machines=machines)
+
+
+def migrate_groups_delete(cmd, client,
+                          resource_group_name,
+                          project_name,
+                          group_name):
+    return client.delete(resource_group_name=resource_group_name, project_name=project_name, group_name=group_name)
+
+
+def migrate_assessments_list(cmd, client,
+                             resource_group_name,
+                             project_name,
+                             group_name=None):
+    if resource_group_name is not None and project_name is not None and group_name is not None:
+        return client.list_by_group(resource_group_name=resource_group_name, project_name=project_name, group_name=group_name)
+    return client.list_by_project(resource_group_name=resource_group_name, project_name=project_name)
+
+
+def migrate_assessments_show(cmd, client,
+                             resource_group_name,
+                             project_name,
+                             group_name,
+                             assessment_name):
+    return client.get(resource_group_name=resource_group_name, project_name=project_name, group_name=group_name, assessment_name=assessment_name)
+
+
+def migrate_assessments_create(cmd, client,
+                               resource_group_name,
+                               project_name,
+                               group_name,
+                               assessment_name,
+                               azure_location,
+                               azure_offer_code,
+                               azure_pricing_tier,
+                               azure_storage_redundancy,
+                               scaling_factor,
+                               percentile,
+                               time_range,
+                               stage,
+                               currency,
+                               azure_hybrid_use_benefit,
+                               discount_percentage,
+                               sizing_criterion,
+                               e_tag=None):
+    return client.create(resource_group_name=resource_group_name, project_name=project_name, group_name=group_name, assessment_name=assessment_name, e_tag=e_tag, azure_location=azure_location, azure_offer_code=azure_offer_code, azure_pricing_tier=azure_pricing_tier, azure_storage_redundancy=azure_storage_redundancy, scaling_factor=scaling_factor, percentile=percentile, time_range=time_range, stage=stage, currency=currency, azure_hybrid_use_benefit=azure_hybrid_use_benefit, discount_percentage=discount_percentage, sizing_criterion=sizing_criterion)

```

Comment:
how about break it into multiple lines for readability?

Reply:
This line too long is in the process of fixing. I think it will be fixed very soon.